pkg/nspawntool/run.go

```diff
@@ -52,17 +53,17 @@ var (
 var k8sbinds []string
 var defaultBinds = []string{
 	// kube-spawn bins
-	"--bind=" + parseBind("$PWD/scripts:/opt/kube-spawn"),
+	bindrw + parseBind("$PWD/scripts:/opt/kube-spawn"),
 	bindro + parseBind("$PWD/kube-spawn-runc:/opt/kube-spawn-runc"),
 	// shared tmpdir
-	"--bind=" + parseBind("$PWD/tmp:/tmp/kube-spawn"),
+	bindrw + parseBind("$PWD/tmp:/tmp/kube-spawn"),
```

Comment:
@iaguis `/tmp/kube-spawn` was already mounted read-write. I just replaced `"--bind="` with `bindrw` to avoid duplications.